dracut/30disk-uuid/disk-uuid@.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Generate new UUID for disk GPT %I
+DefaultDependencies=no
+Wants=local-fs-pre.target
+Before=local-fs-pre.target systemd-fsck-root.service
+Wants=systemd-udevd.service
+After=systemd-udevd.service
+Requires=%i.device
+After=%i.device
+# Run before services that use device nodes, preventing them from racing
+# with udev activity generated by sgdisk
+Before=ignition-setup.service ignition-disks.service
+
+[Service]
+Type=oneshot
```

Comment:
I usually `RemainAfterExit=yes` for `Type=oneshot`, see https://github.com/ostreedev/ostree/pull/1697